lib/system_setting_minibar_metainfo.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+class rex_system_setting_minibar_metainfo extends rex_system_setting
```

Comment:
ich würde den namen ein wenig präziser wählen

```suggestion
class rex_system_setting_minibar_hide_empty_metainfos extends rex_system_setting
```